eng/Directory.Build.Data.props

```diff
@@ -152,4 +152,12 @@
     <!-- Disable doc comments for test projects -->
     <DocumentationFile></DocumentationFile>
   </PropertyGroup>
+
+  <!-- Code generation settings -->
+  <PropertyGroup>
+    <!-- Don't use code generator bundled Azure.Core shared source for code generation -->
+    <AzureCoreSharedCodeDirectory>$(AzureCoreSharedSources)</AzureCoreSharedCodeDirectory>
```

Comment:
I'm not sure I understand this? If the shared sources in the nuget package match the generator why wouldn't we always use those?

Reply:
Generator nuget package carries 2 kinds of shared sources:

1. Generator specific sources
2. Parts of Azure.Core shared that we sync from azure-sdk-for-net to autorest.csharp and are required for generated libraries.

We always use the former from the package but considering this repo has the original versions of Azure.Core shared files we doing used them from the package.